db/seeds.rb

```diff
@@ -5,3 +5,19 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+
+floofers = [
```

Comment:
I'm confused - I thought floofs were pets - why do they have a job title and a current employer?

Reply:
Our floofs are prospective employees for our users. We are doing a fun spin on a LinkedIn but for job seeking pets.